build.gradle

```diff
@@ -48,14 +48,20 @@ project.ext {
 def copyFiles(def projectPath, def exclusions) {
     description 'Copies files into specified build staging location except for noted exclusions'
     copy {
-        from 'LICENSE', 'README.md', 'CONTRIBUTING.md'
+        from 'LICENSE', 'LICENSE.meta'
+        from 'README.md', 'README.md.meta'
+        from 'CONTRIBUTING.md', 'CONTRIBUTING.md.meta'
         from('GooglePlayInstant')
         {
             exclude 'Samples'
         }
         into "${projectPath}/Assets/GooglePlayInstant"
         exclude exclusions
     }
+    copy {
+        from 'GooglePlayInstant.meta'
```

Comment:
I'm not sure we need this meta file. Does a warning still show up in the unity package importer if this is omitted?

Reply:
Yes, it seems to be a problem if I don't match the guid on this folder.